src/main/java/com/CezaryZal/manager/creator/HeadersByToken.java

```diff
@@ -0,0 +1,17 @@
+package com.CezaryZal.manager.creator;
+
+import org.springframework.stereotype.Service;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+
+@Service
+public class HeadersByToken {
+
+    public MultiValueMap<String, String> createHeadersByToken(String token) {
+        String authenticationToken = "Bearer " + token;
+        MultiValueMap<String, String> headers = new LinkedMultiValueMap<>();
+        headers.add("authorization", authenticationToken);
+
+        return headers;
+    }
+}
```

Comment:
Myśle że tego nie powinno być bo request do którego tego używasz powinien zniknąć. A natet jeśli to coś takiego należałoby napisać przez wzorzec Factory. Czyli np klase która ma metode statyczną, która zwraca np RestTemplate (lub dowolny inny klient Http), który ma już wpisany ten token (cośw stylu prefabrykatu obiektu resttemplaete).

Reply:
Obecnie nie jestem wstanie zrobić metody statycznej która budowałaby token w sobie, ponieważ używam spring boota. W klasie ze statyczną metodą tworzącą requesta nie mogę wstrzyknąć klas sewisowych.
Ale samo tworzenie nagłówka i ciała do requesta wrzuciłem do metody statycznej.